libtcmu_config.c

```diff
@@ -479,7 +479,7 @@ struct tcmu_config* tcmu_initialize_config(void)
 	}
 
 	log_dir = getenv("TCMU_LOGDIR");
-	snprintf(cfg->def_log_dir, PATH_MAX, log_dir ? log_dir : TCMU_LOG_DIR_DEFAULT);
+	snprintf(cfg->def_log_dir, PATH_MAX, "%s", log_dir ? log_dir : TCMU_LOG_DIR_DEFAULT);
```

Comment:
Just a small nit. I fixed this to match the existing coding style:

https://github.com/open-iscsi/tcmu-runner/commit/9d8a7e93f9764b5732f0e0745208e42c76ee1f18